### ISSUE TITLE
(Bug): No ratio bar showing

### ISSUE THREAD

**Issue by 3rd3** [Anarios/return-youtube-dislike#522]
### Browser

Firefox

### Browser Version

98.0.2

### Extension or Userscript?

Extension

### Extension/Userscript Version

latest

### Video link where you see the problem

Any

### What happened?

For me this extension never shows the ratio bar.

![Image](https://user-images.githubusercontent.com/7326255/145552036-c7fb65ef-7ca3-42fe-b3d2-b0033ad5f977.png)

### How to reproduce/recreate?

See above.

**Comment by sy-b** [comment]
This is due to YouTube's new UI. You are a part of YT's A/B testing.
You can try clearing cookies (eventually signing-out of YT).

**Comment by cyrildtm** [comment]
Would #495 solve this issue? 
Is this A/B testing consistent with a particular user? Is there a way to force this? 
Oh no. It just looks worse.

**Comment by sy-b** [comment]
> Is there a way to force this?

No, but kind of yes

0. Wish yourself good luck
1. Open browser (for this example I'll be using chromium based browser)
2. Press `ctrl` + `shift` + `n` (chrome) **or** `ctrl` + `shift` + `p`  (Firefox
3. Visit [YoutTube.com](https://youtube.com/)
4. Repeat from step 2 until you get a new YouTube UI

**Comment by cyrildtm** [comment]
Nope. Tried twenty times in a row, turned out step 0 didn't work. 

I just want some A/B testing victim confirm if the container still exists. Since they have removed the dislike, there is no meaning to keep the ratio bar any more. But if it's still there, we may still use it.

**Comment by sy-b** [comment]
Got ratio-bar-less UI on the 6th try.
I can share the HTML (in private chat).

**Comment by cyrildtm** [comment]
got it on 27th try. 

```
x = document.querySelector('ytd-sentiment-bar-renderer#sentiment');
if (x.hasAttribute('hidden')) { x.removeAttribute('hidden');}
y = x.querySelector('div#like-bar')
y.style.width = '50%'
```

This restores ratio bar. It's the same technique as used in #495

```
y = x.querySelector('tp-yt-paper-tooltip div#tooltip')
if (y.classList.contains('hidden')) {y.classList.remove('hidden');}
y.innerHTML = '123/456'
```

This reveals the tooltip, but it's not hover but always on. I don't know how to make it dynamic? In the old UI the `hidden` class is replaced with `fade-in-animation` on mouse hover (and vice versa)

**Comment by databoose** [comment]
These are workarounds however, i think the project should support ratio bars if this UI change is on a significant amount of users, which it is.
